tests/neptune/new/attributes/atoms/test_artifact_hash.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pylint: disable=protected-access
+
+
+from mock import MagicMock
+
+from neptune.new.attributes.atoms.artifact import Artifact
+from neptune.new.internal.operation import AssignArtifact
+from neptune.new.types.atoms.artifact import Artifact as ArtifactVal
+from tests.neptune.new.attributes.test_attribute_base import TestAttributeBase
+
+
+class TestArtifactHash(TestAttributeBase):
```

Comment:
I think you can update `tests.neptune.new.attributes.atoms.test_artifact.TestArtifact` with `AssignArtifact` op instead of creating a whole new test

Reply:
But `TestArtifact` has quite complicated `setUp` and I didn't want to use it for such simple test cases.

Reply:
meh, ok